pkg/features/kube_features.go

```diff
@@ -828,6 +834,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	GracefulNodeShutdown:                           {Default: false, PreRelease: featuregate.Alpha},
 	ServiceLBNodePortControl:                       {Default: false, PreRelease: featuregate.Alpha},
 	MixedProtocolLBService:                         {Default: false, PreRelease: featuregate.Alpha},
+	NamespaceDefaultLabelName:                      {Default: true, PreRelease: featuregate.Beta}, // remove feature gate in v1.22
```

Comment:
fixed

Reply:
graduates to GA and locks to default in 1.22 (at which point no code needs to check it any more). the gate itself is removed in 1.24

Reply:
still outstanding, update this comment to accurately reflect the planned changes/releases

Reply:
solved this